reference/5.0/PowershellGet/Uninstall-Module.md

```diff
@@ -1,5 +1,5 @@
 ---
-ms.date:  06/09/2017
+ms.date: 7/2/2019
```

Comment:
Date needs to be 07/02/2019. Need all the digits. Fix in all articles.

Reply:
Thanks. I updated the date in each document.